coderd/workspaceapps/db.go

```diff
@@ -198,11 +198,9 @@ func (p *DBTokenProvider) Issue(ctx context.Context, rw http.ResponseWriter, r *
 		return nil, "", false
 	}
 
-	// Check that the app is healthy.
-	if dbReq.AppHealth != "" && dbReq.AppHealth != database.WorkspaceAppHealthDisabled && dbReq.AppHealth != database.WorkspaceAppHealthHealthy {
-		WriteWorkspaceAppOffline(p.Logger, p.DashboardURL, rw, r, &appReq, fmt.Sprintf("App health is %q, not %q", dbReq.AppHealth, database.WorkspaceAppHealthHealthy))
-		return nil, "", false
-	}
+	// This is where we used to check app health, but we don't do that anymore
+	// in case there are bugs with the healthcheck code that lock users out of
```

Comment:
```suggestion
	// in case there are bugs with the healthcheck that lock users out of
```

could be a lot of things: misconfiguration, app healthcheck endpoint could be misbehaving